source/configuration_manual/authentication/pam.rst

```diff
@@ -75,12 +77,13 @@ so PAM plugin writers haven't done much testing on what happens when multiple
 lookups are done. Because of this, many PAM plugins leak memory and possibly
 have some other problems when doing multiple lookups. If you notice that PAM
 authentication stops working after some time, you can limit the number of
-lookups done by the auth worker process before it dies:
+lookups done by the auth worker process before it dies using the
+:dovecot_core:ref:`passdb_pam_max_requests` setting:
 
 .. code-block:: none
 
   passdb pam {
-    args = max_requests=100
+    max_requests = 100
   }
```

Comment:
Hmm. I don't think there is any benefit in passdb_pam_max_requests over service auth-worker { restart_request_count }? I think both are doing about the same. So we could likely just remove passdb_pam_max_requests entirely.

Reply:
Although .. I suppose the auth-workers are often used also for userdb lookups. And it's nice that with PAM the process is by default restarting every 100 requests, which can't really be configured as a default with restart_request_count since it would apply to non-PAM configurations also.

So I guess we can keep this setting, but it could refer to the service_restart_request_count setting saying it behaves similarly.

Reply:
I tried to formulate it nicely not sure if it hits the spot